react-charts/bullet-chart/index.json

```diff
@@ -1,6 +1,6 @@
 {
-  "title": "Bullet chart",
-  "description": "Learn how to implement a React bullet chart.",
+  "title": "Bullet Chart",
+  "description": "Learn how to implement a bullet chart using PatternFly React charts.",
```

Comment:
If we change the description here, change it for all charts

Reply:
Reverted for now